rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -461,26 +470,31 @@ def _tag_id_index_mapping_for(
 
         return tag_id_dict
 
-    def _uses_sequence_features_for_text(self) -> bool:
+    def _uses_sequence_features_for_input_text(self) -> bool:
```

Comment:
Please make it static

Reply:
this must stay this way so child classes can overwrite this and use e.g. self.component_config to make this decision

Reply:
Ahh okay.. yes that makes sense.